README.md

```diff
@@ -90,13 +90,65 @@
 ## Getting Started
 
 
-### Prerequisites
-
-
-
-### Installation
-
-
+### Setup
+#### First you need is load Alpine.js and Tailwindcss
```

Comment:
Toto nie je nadpis, to je bod. dajme to ako normalny text. Rovnako si to spravil na vela miestach.

Reply:
Done bro